sympy/functions/elementary/tests/test_hyperbolic.py

```diff
@@ -1097,6 +1097,29 @@ def test_cosh_expansion():
     assert cosh(3*x).expand(trig=True).expand() == \
         3*sinh(x)**2*cosh(x) + cosh(x)**3
 
+def test_cosh_positive():
+    # See issue 11721
+    # cosh(x) is positive for real values of x
+    x = symbols('x')
+    k = symbols('k', real=True)
+    n = symbols('n', integer=True)
+
+    assert cosh(k).is_positive is True
+    assert cosh(k + 2*n*pi*I).is_positive is True
+    assert cosh(I*pi/4).is_positive is True
+    assert cosh(3*I*pi/4).is_positive is False
```

Comment:
You need to use evaluate=False in tests like this because otherwise cosh will evaluate to cos as in
```julia
In [1]: cosh(I*k)                                                                                                                                             
Out[1]: cos(k)
```
so this isn't testing the `cosh._eval_is_positive` method.